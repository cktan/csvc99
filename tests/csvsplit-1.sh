# Test Case : split by -r nrecs 
set -e

rm -f x??
../csvsplit -r 2 in/csvsplit.csv 

for f in x0{0..2}; do
	echo "# File: $f"
	cat $f	
done
