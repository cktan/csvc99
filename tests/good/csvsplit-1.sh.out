# File: x00
1,a
2,b
# File: x01
3,c
4,d
# File: x02
5,e
